docker/entrypoint.sh

```
@@ -0,0 +1,117 @@
+    echo "Starting ${region} on port ${port}"
+
+    # Start binary in background, capturing both stdout and stderr
+    # Use sed to inject region field into JSON lines
+    (
+        export REGION="${region}"
+        export PORT="${port}"
+        exec "${binary}" 2>&1 | while IFS= read -r line; do
+            # Try to inject region into JSON if it looks like JSON
+            if [[ $line =~ ^\{.*\} ]]; then
+                # Insert region field after opening brace
+                echo "$line" | sed 's/{/{"region":"'"${region}"'",/'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)